packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeModal.tsx

```diff
@@ -2,9 +2,22 @@ import styled from 'styled-components';
 import { useSelector } from 'src/hooks/suite';
 import { Modal, Translation } from 'src/components/suite';
 import { UnstakeModalContent } from './UnstakeModalContent';
+import {
+    Card,
+    CollapsibleBox,
+    Column,
+    Grid,
+    H3,
+    List,
+    Row,
+    useMediaQuery,
+    variables,
+} from '@trezor/components';
+import { spacings } from '@trezor/theme';
+import { UnstakingInfo } from 'src/components/suite/StakingProcess/UnstakingInfo';
 
 const StyledModal = styled(Modal)`
```

Comment:
Please use NewModal from `components` like in the StakeModal.

Reply:
added NewModal to unstake, https://github.com/trezor/trezor-suite/pull/14713/commits/aa21e6e49b2370a3e9bbb5035db474d51a1e8519

Reply:
Please move unstake button to the footer of the modal, same as stake button

Reply:
created a separate component for the unstake button, https://github.com/trezor/trezor-suite/pull/14713/commits/03dbbc780d1e2c8bcce13f33e84bcfa328df7e64